SR-IOV forwarder fails with panic on NSMgr restart

### Env description
Packet automation server.

### Steps to reproduce
1. Execute steps from [SR-IOV kernel test](https://github.com/networkservicemesh/deployments-k8s/tree/main/examples/use-cases/SriovKernel2Noop) without cleanup.
2. Restart one of NSMgr pods.

### Actual result
Corresponding forwarder pod restarts with panic:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0xbcf772]

goroutine 17421 [running]:
github.com/networkservicemesh/sdk-sriov/pkg/networkservice/chains/xconnectns.(*sriovServer).Request(0xc000302090, 0xeb8fe0, 0xc00093f650, 0xc000536e60, 0xcb9560, 0xc00093f620, 0xeb8fe0)
	<autogenerated>:1 +0x32
github.com/networkservicemesh/sdk/pkg/networkservice/core/next.(*nextServer).Request(0xc00093f5f0, 0xeb8fa0, 0xc000018a20, 0xc000536e60, 0xc00093f5f0, 0xc0002fa390, 0xea4fe0)
	/go/pkg/mod/github.com/networkservicemesh/sdk@v0.0.0-20210403170232-47fc8b2ac74c/pkg/networkservice/core/next/server.go:69 +0x25a
github.com/networkservicemesh/sdk/pkg/networkservice/core/adapters.(*serverToClient).Request(0xc0003020a0, 0xeb8fa0, 0xc000018a20, 0xc000536e60, 0x0, 0x0, 0x0, 0xc0002fa380, 0xc0002fa2d0, 0x0)
	/go/pkg/mod/github.com/networkservicemesh/sdk@v0.0.0-20210403170232-47fc8b2ac74c/pkg/networkservice/core/adapters/server_to_client.go:45 +0x117
github.com/networkservicemesh/sdk/pkg/networkservice/common/heal.(*healServer).restoreConnection(0xc00033c6e0, 0xeb8f20, 0xc000143700, 0xc0005366e0, 0x0, 0x0, 0x0)
	/go/pkg/mod/github.com/networkservicemesh/sdk@v0.0.0-20210403170232-47fc8b2ac74c/pkg/networkservice/common/heal/server.go:367 +0x385
github.com/networkservicemesh/sdk/pkg/networkservice/common/heal.(*healServer).healAsNeeded(0xc00033c6e0, 0xeb8fe0, 0xc000878720, 0xc0005366e0, 0xc000447140, 0x0, 0x0, 0x0)
	/go/pkg/mod/github.com/networkservicemesh/sdk@v0.0.0-20210403170232-47fc8b2ac74c/pkg/networkservice/common/heal/server.go:218 +0x6da
created by github.com/networkservicemesh/sdk/pkg/networkservice/common/heal.(*healServer).startHeal
	/go/pkg/mod/github.com/networkservicemesh/sdk@v0.0.0-20210403170232-47fc8b2ac74c/pkg/networkservice/common/heal/server.go:161 +0xb9
```

### Expected result
No panics, no restarts.

--- a/pkg/networkservice/chains/xconnectns/server.go
+++ b/pkg/networkservice/chains/xconnectns/server.go
@@ -84,7 +84,7 @@ func NewServer(
 	clientURL *url.URL,
 	clientDialOptions ...grpc.DialOption,
 ) endpoint.Endpoint {
-	rv := sriovServer{}
+	rv := new(sriovServer)
 
 	connectChainFactory := func(class string) networkservice.NetworkServiceServer {
 		return chain.NewNetworkServiceServer(

